R/lmodel2-tidiers.R

```diff
@@ -55,6 +61,7 @@ tidy.lmodel2 <- function(x, ...) {
 
   ret %>%
     inner_join(confints, by = c("method", "term")) %>% 
+    select(-p.value, dplyr::everything()) %>% 
```

Comment:
Why do you need the `select()` line here? What does it do?

Reply:
This changes the order of the columns so p.value is the rightmost column.